http_logs/track.py

```diff
@@ -0,0 +1,5 @@
+def reindex(es, params):
```

Comment:
See the earlier comment about the use of [raw-request](https://esrally.readthedocs.io/en/stable/track.html#raw-request) if you'd like to avoid adding a custom runner.